src/api/planters.js

```diff
@@ -83,12 +83,13 @@ export default {
       order: 'timeUpdated DESC',
       limit: 100,
       fields: ['planterPhotoUrl'],
-      where: { planterId, planterPhotoUrl: { neq: null } },
     };
 
     const treeQuery = `${
```

Comment:
It doesn't make sense to call this `treeQuery` any more, as it's querying planter selfies.
The client doesn't need to know that the api is using the trees database for this lookup.